supervision/utils/video.py

```diff
@@ -192,6 +197,7 @@ def get_video_frames_generator(
     video.release()
 
 
+@deprecated("Use `sv.Video.save()` with a callback for processing and saving videos.")
```

Comment:
We aim to keep the format of all deprecation messages consistent. Please use the message below and ensure the same formatting is applied to all deprecation messages.

```
@deprecated(
    "`process_video` is deprecated and will be removed in "
    "`supervision-0.32.0`. Use `Video().save` instead."
)
```